ui/analyse/src/study/chapterEditForm.ts

```diff
@@ -145,7 +150,21 @@ export function view(ctrl: StudyChapterEditFormCtrl): VNode | undefined {
                 hook: bind(
                   'click',
                   _ => {
-                    if (confirm(noarg('deleteThisChapter'))) ctrl.delete(data.id);
+                    if (confirm(ctrl.trans.noarg('clearSidelines') + '?')) ctrl.clearSidelines(data.id);
```

Comment:
Adding a question mark like this probably won't really work in some languages (it might look weird or just completely wrong). It's better to just leave it as a statement that can be confirmed.